tests/sps30-uart-test.cpp

```diff
@@ -0,0 +1,163 @@
+#include "sensirion_test_setup.h"
+#include "sps30.h"
+
+// Measurement ranges according to datasheet
+#define SPS30_MIN_MC 0
+#define SPS30_MAX_MC 1000
+#define SPS30_MIN_NC 0
+#define SPS30_MAX_NC 3000
+#define CMD_DELAY_USEC 20000
+#define SLEEP_WAKE_UP_DELAY_USEC 5000
+#define SENSIRION_UART_TTYDEV "/dev/ttyUSB1"
```

Comment:
Could we move this into the header wrapped in `#ifndef`?

Reply:
It's just very important that this overwrites any other point where this is defined since otherwise a code change some where else could break the test. So it's important that this define is this and nothing else. From my understanding for this behavior it would be not at all helpful to put this in a `#ifndef`, but please tell me if I am wrong or if there would be a different thing we could do to enforce this behavior.

Reply:
Maybe we need to discuss this in person. The way I see it, is that the build environment should be able to overwrite this with e.g. `make test -D SENSIRION_UART_TTYDEV=/dev/ttyUSB0` - without `#ifndef` this is not possible. If other places hardcode the value, that should probably be questioned, too. Where do you foresee that this could be defined?

Reply:
And this clearly doesn't work as I would like to since it should be USB0 anyway ...

Reply:
the linux sample implementation does define it in a `#ifndef` but yeah if we can pass it as a command line argument that would be best.